src/platform/plugins/shared/unified_search/public/dataview_picker/change_dataview.tsx

```diff
@@ -147,6 +153,18 @@ export function ChangeDataView({
       </EuiButtonEmpty>
     );
   };
+  const onDuplicate = useCallback(async () => {
+    const dataView = await dataViews.get(currentDataViewId!);
```

Comment:
Can we just put an early return clause so `if (!currentDataViewId) return;` rather than asserting it?